requirements.txt

```diff
@@ -1,6 +1,11 @@
-Django==1.10
-django-cors-middleware==1.2.0
-django-extensions==1.7.1
-djangorestframework==3.4.4
-PyJWT==1.4.2
-six==1.10.0
+dj-database-url==0.5.0
```

Comment:
@jomasim you've added **dj-database-url** and I haven't seen any use of it in anywhere in the application, perhaps consider removing it from the requirements file to prevent install of unused dependencies.

Reply:
I was discussing that with @waracci we would probably need to switch to dj-database-url later. warraci was supposed to look into that so will need it later.

Reply:
Is there any advantage in using dj-database-url as opposed to using the normal path urls?

Reply:
@sharkdevs It allows us to only have a single string with all db variables instead of  setting them individually.